src/components/shared/LocationSelect.vue

```diff
@@ -29,20 +34,13 @@ export default {
 </script>
 
 <template>
-  <select
-    v-bind="$attrs"
-    ref="select"
-    @change="handleChange"
-  >
-    <option value="" selected>
-      Şehir seçiniz...
-    </option>
-    <option
-      v-for="location in locations"
-      :value="location"
-      :key="location"
-    >
-      {{ location }}
-    </option>
-  </select>
+  <multiselect
+    v-model="selected"
+    :options="locations"
+    :searchable="false"
```

Comment:
Can we make this a `prop`? Because there will be 82 locations in the add job page. It would be better if we can have search functionality for that specific multiselect.